packages/other/src/Markdown.ts

```diff
@@ -0,0 +1,23 @@
+import { HTMLWidget } from "@hpcc-js/common";
+import * as _marked from "marked";
+
+const marked = _marked;
+
+export class Markdown extends HTMLWidget {
+    constructor() {
+        super();
+    }
+
+    update(domNode, element) {
+        super.update(domNode, element);
+        element.html(marked(this.text()));
+    }
+}
+Markdown.prototype._class += " other_Markdown";
+
+Markdown.prototype.publish("text", "", "string", "Sets text to be rendered as markdown text", null, { tags: ["Basic"] });
+
+export interface Markdown {
+    text(): string;
+    text(_: string): this;
```

Comment:
"text" or "markdown"?  Text is probably ok as markdown is designed to be readable as text also...  (the markdown editor supports both).

Reply:
I went with `text` because it is/was the most intuitive name that came to mind.

Reply:
I think the base class already has a "text" method - in which case (for consistency) you could make this `markdown`?